Week3/hw/app14.js

```diff
@@ -0,0 +1,5 @@
+//  14. Add a property to the object you just created that contains the languages that they have taught you.
+
+// let seenInstructors = ['Philipp', 'Rob', 'Unmesh', 'Bonan', 'Yash'];
+
+seenInstructors.languages = 'HTML, CSS, CLI, GIT, JS1';
```

Comment:
It's good that you've added the languages here as a `property`. But usually it's good practice to use an `array` instead of a `string` for multiple values. So could you change this into an `array`?